sympy/sets/sets.py

```diff
@@ -377,6 +377,28 @@ def is_subset(self, other):
             # This assumes that an unevaluated Intersection will always come
             # back as an Intersection...
             elif isinstance(s_o, Intersection):
+                from sympy import ConditionSet
+                # TODO: add logic for Interval, Union...
+                if isinstance(self, ConditionSet) and isinstance(other, ConditionSet):
```

Comment:
I think it's probably better to add this outside of the `isinstance(Intersection)` branch.